packages/module/src/InvalidObject/InvalidObject.tsx

```diff
@@ -0,0 +1,47 @@
+import { Button, Title } from '@patternfly/react-core';
+
+import Icon404 from './icon-404';
+import React from 'react';
+import classNames from 'classnames';
+import { createUseStyles } from 'react-jss'
+
+const useStyles = createUseStyles({
+  "invalidObject": {
+    display: "flex",
+    justifyContent: "center",
+    flexDirection: "column",
+    alignItems: "center",
+    textAlign: "center",
+    h1: { marginBottom: "30px" },
+    svg: { marginBottom: "30px" },
+    button: { fontSize: "20px" }
+  },
+  "invalidObjectSorryText": { maxWidth: "70%" },
+})
+
+export interface InvalidObjectProps {
+  /** Indicates if an additional "/beta" string should be added to when navigating back to the home screen. */
+  isBeta?: boolean
```

Comment:
I still think rather than the `isBeta` prop we should allow customizing the props it affects.

Reply:
so the whole href should be a prop?

Reply:
@InsaneZein I would vote for being consistent with the `NotAuthorized` component https://github.com/patternfly/react-component-groups/blob/6f42868e3f9bc6c9c1bc1e87833a2da18cf66f54/packages/module/src/NotAuthorized/NotAuthorized.tsx#L53